x-pack/plugins/maps/public/embeddable/map_embeddable.js

```diff
@@ -84,6 +84,13 @@ export class MapEmbeddable extends Embeddable {
     this._store.dispatch(setReadOnly(true));
     this._store.dispatch(setFilterable(true));
 
+    if (_.has(this._embeddableConfig, 'isLayerTOCOpen')) {
+      this._store.dispatch(setIsLayerTOCOpen(this._embeddableConfig.isLayerTOCOpen));
+    } else if (this._savedMap.uiStateJSON) {
+      const uiState = JSON.parse(this._savedMap.uiStateJSON);
+      this._store.dispatch(setIsLayerTOCOpen(_.get(uiState, 'isLayerTOCOpen', true)));
```

Comment:
extract layer-toc default of `true` to constant, as it's used elsewhere as well.